src/users/dto/update-user.dto.ts

```diff
@@ -5,8 +5,7 @@ export class UpdateUserDto {
     @IsNumber()
     id: number;
 
-    @IsNotEmpty()
-    roles: string[];
+    roles?: string[];
```

Comment:
Why did you remove the `@IsNotEmpty()` decorator?

Reply:
For the scenario where the only thing I want to update is the password. With the `@IsNotEmpty()` on roles, the server will throw an error because the user hasn't put a value for roles.